### ISSUE TITLE
[Feature Request] different landscape rotate mode please!

### ISSUE THREAD

**Issue by evianzhow** [henryli17/AlwaysLandscape#1]
Hi Henry,

In your code, there's a fixed value for landscape rotation 'UIInterfaceOrientationLandscapeRight', which rotates the application 90 degrees clockwise. In my use case, I need 'UIInterfaceOrientationLandscapeLeft'. I usually compile these things myself, but I've had no access to my Mac environment lately. Since you are using PreferenceLoader, is it possible to add an option in the panel, letting the user choose between Left or Right?

**Comment by henryli17** [comment]
Hey, I don't have Theos setup on my machine anymore - but you no longer need a Mac to compile https://theos.dev/docs/installation-linux

**Comment by evianzhow** [comment]
Hi Henry,

Thanks for the information. I've set up the Theos development environment on my Windows computer using WSL 2 and successfully compiled. However, I noticed that the version you uploaded on Cydia was different from this source code.

I encountered issues with losing WiFi and USB connection to my iPhone after an iOS restart when self-compiling with your source code. Eventually, I found that it may be related to a tweak interfering with Apple's internal Wi-Fi management processes, causing malfunctions. It turns out that this line (https://github.com/henryli17/AlwaysLandscape/blob/37a6f01b4a094bb5aca2c9648495f99c1b63ec83/Tweak.x#L18C28-L18C28) was unsafe because some processes might return nil. After switching to square brackets for calling type and adding some if-else statements, this problem was solved.

This is a friendly notification; you might want to modify your code as you see fit.
